xAPISGProcessor.ipynb

```diff
@@ -354,6 +354,56 @@
     "%run vis/xAPISG-AlternativesSelectedQuestion.ipynb\n",
     "vis_alternatives_selected_question(players_info)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "**Bar player, with number of times each item has been interacted by**"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "%run vis/HelpersFunctions/MultiBarSeparated.ipynb\n",
+    "%run vis/xAPISG-ItemsInteracted.ipynb\n",
+    "vis_items_interacted_times(players_info)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "**Heatmap, for each player, the number of times each item has been interacted by**"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "%run vis/HelpersFunctions/HeatMap.ipynb\n",
```

Comment:
Helper functions could better be run in the visualization notebook, to hide lower-level details here.